SourceCpp/React.cpp

```diff
@@ -300,8 +298,8 @@ PeleC::react_state(
           } else {
 #ifdef AMREX_USE_GPU
             react(
-              bx, rhoY, frcExt, T, rhoE, frcEExt, fc, mask, dt, current_time,
-              reactor_type, amrex::Gpu::gpuStream());
```

Comment:
just a minor comment, i would like to keep the reactor_type variable, just to be clear on the arguments on the call.